tests/test_persisted_scale_property.py

```diff
@@ -0,0 +1,21 @@
+"Contains a collection of pytest tests that validates the persisted scale properties."
+
+import nidaqmx
+import nidaqmx.system.system
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestPersistedScaleProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the persisted scale properties."""
+
+    def test_bool_property(self):
+        """Test for validating bool attributes in persisted scale."""
+        system = nidaqmx.system.System.local()
+        persisted_scale = system.scales[0]
+        assert persisted_scale.allow_interactive_editing
```

Comment:
@zhindes Will `allow_interactive_editing` be always true, irrespective of the instrument tested?

Reply:
This property is based on one of the inputs to `DAQmx Save Scale`: https://www.ni.com/docs/en-US/bundle/ni-daqmx-labview-api-ref/page/lvdaqmx/mxsavescale.html